internal/dataplane/parser/translate_udproute_test.go

```diff
@@ -645,7 +942,8 @@ func TestIngressRulesFromUDPRoutesUsingExpressionRoutes(t *testing.T) {
 					routeName := expectedRoute.Name
 					r, ok := kongRouteNameToRoute[*routeName]
 					require.Truef(t, ok, "should find route %s", *routeName)
-					require.Equalf(t, expectedRoute.Expression, r.Expression, "route %s should have expected expression", *routeName)
+					// require.Equalf(t, expectedRoute.Expression, r.Expression, "route %s should have expected expression", *routeName)
+					require.Equalf(t, expectedRoute.Expression, r.Expression, "route %s should have expected expression >>> expected: %q >>> actual: %q", *routeName, *expectedRoute.Expression, *r.Expression)
```

Comment:
The `require` framework should print the provided objects when the assertion fails. Why provide them in the args?

Reply:
This is leftover from the debugging, I removed this. Unfortunately, `require` in spite of compared dereferenced values for equality does not dereference when printing them, it prints addresses. This is why I introduced those lines during debugging